polygeist/tools/cgeist/Test/Verification/sycl/math_funcs.cpp

```diff
@@ -0,0 +1,86 @@
+// RUN: clang++ -Xcgeist --use-opaque-pointers=1 -fsycl -fsycl-targets=spir64-unknown-unknown-syclmlir -O0 -w %s -o %t.O0.out 2>&1 | FileCheck %s --allow-empty --implicit-check-not="{{error|Error}}:"
```

Comment:
I'd also like to have tests to check that the `sycl` operations are being generated in the first place (using `MLIR` as FileCheck prefix is the standard here)